### ISSUE TITLE
ENS name not defined

### ISSUE THREAD

**Issue by rstormsf** [MetaMask/metamask-plugin#1731]
```
popup.js:79375 Error: ENS name not defined.
    at Object.257../abis/registry.json (popup.js:28679)
    at s (popup.js:1)
    at popup.js:1
    at Object.<anonymous> (popup.js:79281)
    at Object.669.babel-runtime/core-js/object/keys (popup.js:79447)
    at s (popup.js:1)
    at popup.js:1
    at Object.707.../lib/persistent-form (popup.js:83801)
    at s (popup.js:1)
    at popup.js:1
```

I Keep getting this error whichever name I'm trying to paste.
`alex.miller.eth`, `vitalik.eth` `romanstorm.eth` 
Doesn't work.
Resolves perfectly fine by https://registrar.ens.domains/#romanstorm

**Comment by flyswatter** [comment]
MetaMask respects the ENS resolver paradigm, and will not resolve a name unless it has a resolver configured. `romanstorm.eth` is owned by `0039f22efb07a647557c7c5d17854cfd
6d489ef3`, but that owner hasn't configured a resolver. `vitalik.eth` is owned by `d8da6bf26964af9d7eed9e03e53415d3
7aa96045` but the owner hasn't set a resolver, either. `alex.miller.eth` would require a subdomain of `miller.eth`, but that name is too short for the current ENS resolver.

To see what a correctly set up name looks like, try sending ether (lots, if you'd like ;)  ) to `dinodan.eth`.

We used to support sending to name owners if no registrar was set up, but we got objection from the ENS creators, and were afraid we would establish some antipattern precedent if we permitted it for too long.
